### DISCUSSION TITLE
Blocked aria-hidden on a <a> element because the element that just received focus must not be hidden from assistive technology users.

### DISCUSSION THREAD

**Discussion by Mannjunath** [primefaces/community#2634] [PrimeVue]
Betting below mentioned console error for Menu component items 
Blocked aria-hidden on a <a> element because the element that just received focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at

**Comment by GonzaloGPF** [comment]
I'm facing the same error. When clicking in Menubar's items I'm getting following error
![menubar-error](https://github.com/user-attachments/assets/fdd38e50-056a-4997-a2c0-e7920f6bd382)
